Source/WebCore/storage/StorageNamespaceProvider.h

```diff
@@ -59,8 +62,8 @@ class StorageNamespaceProvider : public RefCounted<StorageNamespaceProvider> {
 
 private:
     friend class Internals;
-    WEBCORE_EXPORT StorageNamespace& localStorageNamespace(PAL::SessionID);
-    StorageNamespace& transientLocalStorageNamespace(SecurityOrigin&, PAL::SessionID);
+    WEBCORE_EXPORT StorageNamespace& localStorageNamespace(PAL::SessionID, unsigned quota = defaultLocalStorageQuotaInBytes);
+    StorageNamespace& transientLocalStorageNamespace(SecurityOrigin&, PAL::SessionID, unsigned quota = defaultLocalStorageQuotaInBytes);
 
```

Comment:
Please remove the default values for the parameters. The calls lo localStorageNamespace and transientLocalStorageNamespace should always receive a valid value that comes from the setting you added.

Reply:
Sure, will remove it. I added this because without the default arg value, I had to update a test file which still is calling localStorageNamespace() with one arg. I think, I can update that call itself with second arg and get rid of this default arg.